/**: # namespace and node name
  ros__parameters:
    num_waypoints: 1
    waypoint_1: [8.58, 2.5]
    # waypoints: [[8.58, 2.5], [2.5, 8.58], [2.5, 2.5], [8.58, 8.58]]